Policies/amendment-to-github-terms-of-service-applicable-to-u-s-federal-government-users.md

```diff
@@ -7,11 +7,11 @@ redirect_from:
 
 {{#tip}}
 
-This Amendment to GitHub's [Terms of Service](/articles/github-terms-of-service) applies only to users that are using GitHub on behalf of a United States federal government agency. If you are not using GItHub on behalf of a U.S. federal government agency, the standard [GitHub Terms of Service](/articles/github-terms-of-service) applies to you.
+This Amendment to GitHub's [Terms of Service](/articles/github-terms-of-service) applies only to users that are using GitHub on behalf of a United States federal government. If you are not using GItHub on behalf of a U.S. federal government agency, the standard [GitHub Terms of Service](/articles/github-terms-of-service) applies to you.
```

Comment:
Figured it out, I think.

"a United States federal government" should be "the United States federal government"